chapters/data-processing.tex

```diff
@@ -0,0 +1,783 @@
+Finally, each variable will contain data points for different individuals in a given unit of observation.
+In our example, each household member is a subject, and the unit of observation is a person.
+The collection of all data points for each individual subject is an 
+\textbf{observation}.\sidenote{\textbf{Observation:} 
+	the collection of all data points representing attributes of the same individual or unit.}
+
+% Tidy data
+The ideal format to interact with any table or dataset is when
+each column represents one variable,
+each row represents one observation, 
+and all variables in a table have the same unit of observation.
+Datasets in this format are said to be \textbf{tidy},\sidenote{\textbf{Tidy data:}
+	Datasets where each column represents one variable,
+	each row represents one observation, 
+	and all variables in a table have the same unit of observation.}
+and all others are \textit{untidy}.
+This may seem trivial, but the raw data acquired is frequently untidy.
+Tidy data sets are easier to handle,
+as observations can be aggregated and linked consistently,
+and easier to clean, as each attribute only needs to be checked once.
+
+% An example of untidy data
+A very simple way for our survey example to became untidy is to include
+household-level variables in the same data set,
+such as whether the household has a refrigerator.
+In this case, some attributes would be measured at
+household level, and others at household member level,
+and it would not be possible for each column to represent a variable
+\textit{and} for each row to represent an observation \textit{at the same time}.
+Either the household-level variable would be repeated for all its members,
+or member-level variables would be stored in 
+\textbf{wide format}.\sidenote{\textbf{Wide data:}
+	a table where a single variable is divided into multiple columns,
+	for example one for each individual.
+}
+An alternative example is a dataset on transactions that also contains information
+about the parts involved in each transaction on the same table.
+The next two sections will outline steps to check if your data is tidy,
+and to tidy it if it is not.
+
+\subsection{Tidying raw data}
+
+% Is the uniquely identified data tidy?
+Sometimes, making sure that data is uniquely and fully identified 
+is enough to yield a tidy dataset.
+This happens when a data source or survey instrument
+contains a single unit of observation.
+If you are working with data in this format,
+you may skip the tidying process,
+which can become quite intricate.
+In this case the cleaning process will result in a single dataset.
+
+% Examples of untidy raw data
+The most common case of untidy raw data encountered in development research
+are datasets consisting of multiple units of observation stored in the same table. 
+If your rows include multiple nested observational units,
+having an identifying variable for each row doesn't mean having an identifier for each observation.
```

Comment:
I think this sentence should come sooner as I think this is the core of the philosophy of tidy. That one observation often have sub-observations, and tidy data is to put sub-observations in their own table, where they are the main observation, while still keeping a link to the partent observation.

Reply:
done in d2cd334